src/game/editor/editor.cpp

```diff
@@ -658,7 +658,7 @@ void CEditor::DoToolbar(CUIRect ToolBar)
 	// proof button
 	TB_Top.VSplitLeft(40.0f, &Button, &TB_Top);
 	static int s_ProofButton = 0;
-	if(DoButton_Editor(&s_ProofButton, "Proof", m_ProofBorders, &Button, 0, "[ctrl+p] Toggles proof borders. These borders represent what a player maximum can see.") ||
```

Comment:
I am not an english native so I can not give any feedback on how the sentence improved. But technically speaking "maximum" does make more sense to me than "how far". Since users have different screen resolutions the view port varies. The proof border helps the mapper to make stuff big enough so it looks good on all sizes. But there is no guarantee that these borders represent "how far" users can see. Its more how far they could potentially see.

Reply:
Okay, got it